### DISCUSSION TITLE
Welcome to discussions!

### DISCUSSION THREAD

**Discussion by hteumeuleu** [hteumeuleu/email-bugs#87] [General]
Welcome to discussions on email-bugs!

I’ve been eager to activate this new feature on this repo since it was announced earlier this year. While I enjoy the instantaneity of Slack and Twitter conversations, I strongly believe it’s important to have a place where we can see older conversations and link to them. So let’s try this and see how it goes.

Feel free to start discussions about specific HTML emails problems you're encountering, about your latest finding of changes in email clients or about your latest tricks you've used in your HTML emails code.

Please stay civilized and courteous towards each others here and there. (Remember that even the worst email clients are made by human beings, too.)

Let me know if you've got any ideas or suggestions for this place.

**Comment by jroobz** [comment]
the unsubscribe link is broken.
please fix and unsubscribe me, thanks

**Reply by hteumeuleu** [reply]
That's all part of GitHub, I have no idea how I would do that.

**Reply by revelt** [reply]
@jroobz did you try this option at the top:
![Screenshot 2022-09-08 at 13 07 27](https://user-images.githubusercontent.com/8344688/189118353-6ad29a7e-6a60-488b-9167-fdd9c077d715.png)
